proxy/envoy/config.go

```diff
@@ -278,6 +282,21 @@ func buildOutboundHTTPRoutes(instances []*model.ServiceInstance, services []*mod
 				glog.Warningf("Unsupported outbound protocol %v for port %#v", protocol, servicePort)
 			}
 		}
+
+		clusters.setTimeout(context.MeshConfig.ConnectTimeout)
+
+		// apply SSL context to outbound clusters for authentication policy
+		switch context.MeshConfig.AuthPolicy {
+		case proxyconfig.ProxyMeshConfig_NONE:
+		case proxyconfig.ProxyMeshConfig_MUTUAL_TLS:
+			serviceAccounts, _ := context.Discovery.GetIstioServiceAccounts(service.Hostname)
```

Comment:
Do we need to throw some exception or add log something when it fails to get istio service ?